.backportrc.json

```diff
@@ -0,0 +1,7 @@
+{
+  "upstream": "elastic/beats",
+  "branches": [{ "name": "7.9", "checked": true }, { "name": "7.8"}, { "name": "7.7"}, { "name": "7.x"}],
```

Comment:
Why 7.9? Developers are backporting to 7.x only. We will cut 7.9 after FF

Reply:
>Why 7.9? 

it should not be there, I've deleted it.

>Developers are backporting to 7.x only. We will cut 7.9 after FF
7.8.0 is already released. There will be no more backports to 7.7

We (Team:Automation) have to backport any change on the Jenkinsfiles to the active branches to avoid lose functionality or fix bugs. When the tool is launched you have to select the branch/es to backport to so I think is fine, developers should only select 7.x.